Sources/Sentry/Public/SentryOptions.h

```diff
@@ -264,6 +264,16 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL enableFileIOTracing;
 
+/**
+ * Indicates whether tracing should be enabled.
+ * Enabling this will change `tracesSampleRate` to 1 if both
+ * `tracesSampleRate` and `tracesSampler` are nil.
+ *
+ * Changing either `tracesSampleRate` or `tracesSampler` to
+ * a value other then nil will enable this in case this was never changed before.
```

Comment:
What if the user disabled the tracing on purpose, then change the rate, but plans to enable the tracing in a later moment?

By changing the rate after disabling tracing it would enable it again and that's why I think we should use `_enableTracingManual`.

Reply:
I agree. Changes in the `tracesSampleRate` or `tracesSampler` should not impact this property.